PurchasesCoreSwift/Misc/DangerousSettingOption.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  DangerousSettingOption.swift
+//  PurchasesCoreSwift
+//
+//  Created by Cesar de la Vega on 1/25/22.
+//  Copyright © 2022 Purchases. All rights reserved.
+//
+
+/**
+ Only use a Dangerous Setting if suggested by RevenueCat support team.
+ */
+@objc(RCDangerousSettingOption) enum DangerousSettingOption: Int {
+
```

Comment:
This assumes that all potential future settings will just be on or off. I hope it doesn't bite us?
Otherwise adding more options here that are only compatible for some settings seems like the wrong API.
I guess I'm used to Swift only where we could represent this better with an enum and associated values, but I don't have a better suggestion for Objective-C.

Reply:
I am not happy about this either. How would you do the enum with associated values? I am not sure I know how to represent that.

For now the plan is to release this as objc for a one off release for some developers, then add it to v4, which will be in Swift.

Reply:
But even in Swift doing that wouldn't be visible to Objective-C so we're stuck with this option.

Reply:
Right. I updated to a class, which I think makes more sense, since we can type the values. If it grows too much in the future, we can rethink this.